boards/arm/nrf9131ek_nrf9131/nrf9131ek_nrf9131.yaml

```diff
@@ -0,0 +1,21 @@
+identifier: nrf9131ek_nrf9131
+name: nRF9131-EK-NRF9131
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+ram: 88
```

Comment:
88KB of RAM?

Reply:
should I put 128kB of usable ram (for non-secure) or 256kB of total SoC ram?

Reply:
256 probably since tests can use the full 256KiB (I presume?)